src/main/java/org/folio/edge/rtac/utils/RtacOkapiClient.java

```diff
@@ -35,10 +37,10 @@ protected RtacOkapiClient(Vertx vertx, String okapiURL, String tenant, long time
         tenant,
         requestBody,
         combineHeadersWithDefaults(headers),
-        response -> response.bodyHandler(body -> {
+        response -> {
           int statusCode = response.statusCode();
-          if (statusCode == 200) {
-            String responseBody = body.toString();
+          String responseBody = response.body().toString();
+          if (statusCode == 200) {    
             logger.info(String.format(
                 "Successfully retrieved title info from mod-rtac: (%s) %s",
                 statusCode,
```

Comment:
Structure of responses has changed slightly in vertx 4, requiring some refactoring.